### ISSUE TITLE
so lost (working on variables in Javascript)

### ISSUE THREAD

**Issue by seancashmere** [nodeschool/discussions#1379]
Hi. No clue as to why this will not work. Clearly I have no idea what I'm doing. Don't even know how to search to see if this was asked prior.

Here's my code typed in Atom on a Mac:

var example = 'some string';
console.log(example);

I do have a file (in Atom) called "variables.js" in a directory known as javascripting. When I type "javascripting verify variables.js" I get:
# O-oh, something isn't working.
# But don't panic!

---
## Check your solution:
# Solution

some string
# Your Attempt
# Difference

some string
## Additional troubleshooting:
- Did you type the name of the file correctly? You can check by running ls variables.js, if you see ls: cannot access variables.js: No such file or directory then you should create new file / rename existing or change directories to the one with file
- Make sure you didn't omit parens, since otherwise compiler would not be able to parse it
- Make sure you didn't do any typos in the string itself

Need help? Ask a question at: github.com/nodeschool/discussions/issues

It does not make any sense why what I wrote does not work.

**Comment by cerkiewny** [comment]
Hello, I suggest you read tutorial one more time, it looks like the string you are typing into the console doesn't match the one you should be putting.

Best Regards

**Comment by cerkiewny** [comment]
Your code is fine, it works for me. It looks like you have something wrong with a console.
Try the command "node variables.js" you should get the "some string" in the console.

**Comment by seancashmere** [comment]
Try it where? In terminal? Or in atom? What do you mean something is wrong with a console? Again, I am super new. I'm also outside of Seattle and there's a meet-up tomorrow evening I'm going to go to to try and get better. It'd be nice to have some knowledge ready though.

So what could possibly have gone awry with my setup?

Thanks for getting back!

**Comment by cerkiewny** [comment]
Try it in terminal. I am not sure what do you mean by atom, do you mean the text editor? If yes then I don't know how it is executing maybe its creating its own virtual console in graphical mode and claiming the string you are outputting.

tl;dr: try it through console.
